model/textparse/promlex.l

```diff
@@ -27,6 +27,9 @@ const (
     sLValue
     sValue
     sTimestamp
+    sExemplar
+    sEValue
+    sETimestamp
```

Comment:
Examplars are not supported by the prometheus text format

Reply:
But they are needed for the model/textparse/openmetricslex.l.go and the consts cannot be declared twice in the same package . How should it be handled ?

Reply:
ooh I see

Reply:
it seems we have diverted from the auto generated files